config/samples/dataplane_v1beta1_openstackdataplane_baremetal_with_ipam.yaml

```diff
@@ -89,12 +102,11 @@ spec:
           edpm_nova_compute_container_image: "{{ registry_url }}/openstack-nova-compute:{{ image_tag }}"
           edpm_nova_libvirt_container_image: "{{ registry_url }}/openstack-nova-libvirt:{{ image_tag }}"
           edpm_ovn_metadata_agent_image: "{{ registry_url }}/openstack-neutron-metadata-agent-ovn:{{ image_tag }}"
-
           gather_facts: false
           enable_debug: false
           # edpm firewall, change the allowed CIDR if needed
           edpm_sshd_configure_firewall: true
-          edpm_sshd_allowed_ranges: ['192.168.111.0/24']
```

Comment:
This had .111 as normally folks would deploy with dev-scripts deployed OCP cluster with CBO. We can change it though.

Reply:
since this value is patched by install_yamls scripts anyway, leaving the defaults to match dev-scripts might be best